Prototypes/MorganStanley.ComposeUI.Prototypes.NoPluginsPrototype/MorganStanley.ComposeUI.Prototypes.NoPluginsPrototype.sln

```diff
@@ -0,0 +1,128 @@
+﻿
```

Comment:
The purpose of this solution is to be copied when one wants to create another solution with plugins. Just to make it easier to churn the prototype solution.

Reply:
I see, if this is reproduced in another PR then it would be helpful to put a readme.md that states this.  Perhaps also name it something like TemplateSolution not NoPluginsPrototype